hack/verify-flags/known-flags.txt

```diff
@@ -12,6 +12,7 @@ allowed-not-ready-nodes
 anonymous-auth
 api-advertise-addresses
 api-external-dns-names
+api-port
```

Comment:
I think it's should be sorted, yes.

And I do see other kubeadm flags there as well :)